### ISSUE TITLE
Ports on the top or bottom of bug render incorrectly

### ISSUE THREAD

**Issue by KorvinSzanto** [tscircuit/tscircuit#120]
Here's a minimal example board:

```tsx
export const Foo = () => <bug port_arrangement={{top_size: 2}} port_labels={{1: 'foo', 2: 'baz'}} />
```

| Side | Example |
|--------|--------|
| Top | ![image](https://github.com/tscircuit/tscircuit/assets/1007419/1dd0df39-e4f6-4222-a29c-34e10a10f7f4) |
| Bottom | ![image](https://github.com/tscircuit/tscircuit/assets/1007419/e1e312e4-5a13-4de0-bc5c-40616443437f) |
| Left | ![image](https://github.com/tscircuit/tscircuit/assets/1007419/d14a69d7-ddc3-4324-8204-9f28f2aecb27) |
| Right | ![image](https://github.com/tscircuit/tscircuit/assets/1007419/91890da5-b8d6-4094-a873-80562e66b92f) | 

Note that for left and right the red dashes line up with the blue boxes whereas for top and bottom they do not.

**Comment by KorvinSzanto** [comment]
I noticed everything is upside down when using `<Schematic>...</Schematic>` from `@tscircuit/schematic-viewer1` compared to the orientation in `tsci dev`, perhaps that's related

**Comment by seveibar** [comment]
Yea recently I flipped the Y axis so that positive was up, this has had a lot of second-order consequences. Will get this fixed, thanks for reporting!

**Comment by seveibar** [comment]
Hey @KorvinSzanto , I _think_ this should be fixed now with @tscircuit/schematic-viewer@1.2.4 and @tscircuit/builder@1.5.88. Note that the builder is a peer dependency of the schematic-viewer.

![image](https://github.com/tscircuit/tscircuit/assets/1910070/16cd35ab-2f18-4501-9337-19fa95e8ebcc)
